### PR TITLE
adds portforward privilege for cssre

### PR DESCRIPTION
**Pull request by chopraapooja**

### CODE REVIEW

File: deploy/backplane/cssre/rhmi/01-rhmi-cssre-admins-project.ClusterRole.yaml [hunk 1]
```diff
@@ -10,6 +10,14 @@ rules:
   - pods/exec
   verbs:
   - create
+# CS SRE can portforward pods
```
Comment (sedroche) [inline deploy/backplane/cssre/rhmi/01-rhmi-cssre-admins-project.ClusterRole.yaml:13]: We don't want to grant permissions in all namespaces. Can you create a new separate ClusterRole for these perms under `deploy/backplane/cssre/`

then update the relevant subjectpermission for each addon:
https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/cssre/rhmi/02-rhmi-cssre-admins.SubjectPermission.yml#L7
https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/cssre/rhoam/02-rhoam-cssre-admins.SubjectPermission.yml#L7

namespaces listed here - https://issues.redhat.com/browse/OSD-7398?focusedCommentId=16351212&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16351212
`redhat-rhmi-*` in rhmi subjectpermission and `redhat-rhoam-*` in rhoam subjectpermission

See rbac-permissions-operator - https://github.com/openshift/rbac-permissions-operator#subjectpermission-cr

**Comment by wanghaoran1988** [comment]
@chopraapooja Could you manually run "make" locally to commit the changes? not sure why the commit is not created automatically.

**Pull request merged**